dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -1517,6 +1517,97 @@ sources:
                   config: *unique-conditions
                   meta:
                     description: yrblt should be between 1850 and now
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_205_more_than_62_years_old
+                  min_value: 1850
+                  max_value: cast(year(current_date) as decimal) - 62
+                  additional_select_columns: *select-columns
+                  config: 
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '205'
+                  meta:
+                    description: 205s should be > 62 years old
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_206_more_than_62_years_old
+                  min_value: 1850
+                  max_value: cast(year(current_date) as decimal) - 62
+                  additional_select_columns: *select-columns
+                  config: 
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '206'
+                  meta:
+                    description: 206s should be > 62 years old
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_207_less_than_62_years_old
+                  min_value: cast(year(current_date) as decimal) - 62
```

Comment:
Both min and max are coded as 62. By the class definitions, it would seem as though the minimum should be 62 (up to 62 years old) and when it is max, it should be 63 (over 62 years old). I don't know how accurately these transitions occur, and trying to accurately code these may cause unexpected spikes.

Reply:
**[Suggestion, non-blocking]** This is a good question, I'm not sure which of the two ranges should include dwellings that are exactly 62 years old. I've escalated this question, but for now I would interpret the class definition language to indicate that "up to 62 years of age" includes 62-year-old dwellings while "over 62 years of age" excludes them.